### PR TITLE
docs: minor

### PR DESCRIPTION
**Pull request by phonzammi**

### CODE REVIEW

**Comment by brillout** [comment]
Let's put the link in a `## See also` section. Self-merge then.

**Comment by brillout** [comment]
Also let's move the `CHANGELOG.md` links to the appropriate docs. Intervention++.

**Comment by phonzammi** [comment]
> Also let's move the `CHANGELOG.md` links to the appropriate docs

Sorry, I don't get it

**Comment by brillout** [comment]
Let's add a link to the CHANGELOG somewhere in https://github.com/vikejs/vike-react/tree/main/packages/vike-react-apollo#readme. So far I'm thinking we can simply add it to `## See also`, but alternative ideas welcome.

File: packages/vike-react-query/README.md [hunk 1]
```diff
@@ -436,6 +436,8 @@ Upon page navigation (and rendering the first page if [SSR is disabled](https://
 
 ## See also
 
+- Docs: [vike.dev/tanstack-query](https://vike.dev/tanstack-query)
+- Version history: [CHANGELOG.md](CHANGELOG.md)
```
Comment (brillout) [inline packages/vike-react-query/README.md:440]: Relative links will be broken on npm.

File: README.md [hunk 2]
```diff
@@ -1,17 +1,4 @@
...
-  - Source code: [packages/vike-react/](packages/vike-react)
-- `vike-react-query` ([TanStack Query](https://tanstack.com/query) integration)
-  - Docs: [README.md](packages/vike-react-query#readme)
-  - Version history: [CHANGELOG.md](packages/vike-react-query/CHANGELOG.md)
-  - Source code: [packages/vike-react-query/](packages/vike-react-query)
-- `vike-react-apollo` ([Apollo GraphQL](https://www.apollographql.com/docs/react) integration)
-  - Docs: [README.md](packages/vike-react-apollo#readme)
-  - Version history: [CHANGELOG.md](packages/vike-react-apollo/CHANGELOG.md)
-  - Source code: [packages/vike-react-apollo/](packages/vike-react-apollo)
+Monorepo of [Vike React extensions](https://vike.dev/extensions#react)
```
Comment (brillout) [inline README.md]: Trailing dot is missing. Intervention++.

**Comment by brillout** [comment]
Did some polishing. If you agree with all of it then please self-merge.

**Pull request merged**